webapp/components/ListItem/ListItem.vue

```diff
@@ -1,17 +1,20 @@
 <template>
   <div id="list-item">
     <h2>{{ item.title }} - ({{ item.votes }})</h2>
-    <button @click="upvoteItem()">Upvote</button>
-    <button @click="downvoteItem()">Downvote</button>
-    <button @click="$emit('delete-item', item)">Remove</button>
+    <button v-if="isAuthenticated" @click="upvoteItem()">Upvote</button>
```

Comment:
⭐ For an upvote button that behaves according to the authentication state of your user